openpathsampling/experimental/simstore/storage.py

```diff
@@ -384,11 +382,11 @@ def load(self, input_uuids, allow_lazy=True, force=False):
         logger.debug("Getting internal structure of %d non-cached objects",
                      len(uuid_list))
         to_load, lazy_uuids, dependencies, uuid_to_table = \
-                get_all_uuids_loading(uuid_list=uuid_list,
-                                      backend=self.backend,
-                                      schema=self.schema,
-                                      existing_uuids=self.cache,
-                                      allow_lazy=allow_lazy)
+            get_all_uuids_loading(uuid_list=uuid_list,
+                                  backend=self.backend,
+                                  schema=self.schema,
+                                  existing_uuids=self.cache,
+                                  allow_lazy=allow_lazy)
```

Comment:
Not urgent, but we have an IDE mismatch thing here. Your IDE says that backslash continuation gets one level of indentation; mine says two levels. In general, I like the 2-level indentation because I feel like backslash continuation is different from other indentation (like parentheses), mainly because backslashes should be avoided if possible. Can you check if that's a setting that you can modify?

(On the other hand, your IDE does a kind of block-comment thing that mine does not -- that approach is better, and I should try to figure out if I can start doing that. I hope that's just the difference between be typing `,c ` and `,cs` -- that was the difference for C/C++ code, at least).

Reply:
I have no strong preference here, so will revert this change